core/gear_log/file_handle.ex

```diff
@@ -84,20 +85,16 @@ defmodule AntikytheraCore.GearLog.FileHandle do
     String.replace(base_file_path, ~R/gz\z/, now_str_with_ext)
   end
 
-  if Antikythera.Env.compiling_for_release?() do
-    defunp write_debug_log(_ :: Level.t, _ :: String.t) :: :ok, do: :ok
-  else
-    defunp write_debug_log(level :: v[Level.t], formatted :: v[String.t]) :: :ok do
-      if Mix.env() == :dev do
-        case level do
-          :error -> IO.ANSI.red() <> formatted <> IO.ANSI.reset()
-          :debug -> IO.ANSI.cyan() <> formatted <> IO.ANSI.reset()
-          _      -> formatted
-        end
-        |> IO.binwrite()
-      else
-        :ok # don't put logs during tests
-      end
+  defunp write_debug_log(level :: v[Level.t], formatted :: v[String.t]) :: :ok do
+    case level do
+      :error -> IO.ANSI.red() <> formatted <> IO.ANSI.reset()
+      :debug -> IO.ANSI.cyan() <> formatted <> IO.ANSI.reset()
+      _      -> formatted
     end
+    |> IO.binwrite()
+  end
+
+  def determine_write_to_terminal() do
```

Comment:
```suggestion
  defp determine_write_to_terminal() do
```

Reply:
If you don't apply my suggestion, please tell me why.

Reply:
I'm sorry, it's an operation mistake